### PR TITLE
KEYCLOAK-13104 Signed and Encrypted ID Token Support : AES 192bit and 256bit key support

### PR DESCRIPTION
**Pull request by tnorimat**
This PR is for [KEYCLOAK-13104](https://issues.redhat.com/browse/KEYCLOAK-13104) .

### CODE REVIEW

**Comment by tnorimat** [comment]
The tests failed but not relevant to this documentation PR.

```
Running org.keycloak.documentation.test.ServerAdminTest
Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 165.892 sec <<< FAILURE!
checkExternalLinks(org.keycloak.documentation.test.ServerAdminTest)  Time elapsed: 165.67 sec  <<< FAILURE!
* http://www.haproxy.org/#docs (exception Connect to www.haproxy.org:80 [www.haproxy.org/51.15.8.218] failed: Connection timed out (Connection timed out))
```

**Review by mposolda — DISMISSED** [review]
@andymunro Do you have a chance to review this please?

File: server_admin/topics/clients/client-oidc.adoc [hunk 1]
```diff
@@ -187,11 +187,11 @@ Key Encryption's algorithms are defined in https://tools.ietf.org/html/rfc7518#s
 * open `Advanced Settings`
 * select `RSA1_5` or `RSA-OAEP` from `ID Token Encryption Key Management Algorithm` pulldown menu
 
-ID token encryption algorithms by CEK are also defined in https://tools.ietf.org/html/rfc7518#section-5.1[JWA] specification. {project_name} supports AES_128_CBC_HMAC_SHA_256 authenticated encryption (A128CBC-HS256) and AES GCM using 128-bit key (A128GCM). The detailed procedure to select this algorithm is as follows:
+ID token encryption algorithms by CEK are also defined in https://tools.ietf.org/html/rfc7518#section-5.1[JWA] specification. {project_name} supports AES_CBC_HMAC_SHA2 algorithms and AES GCM algorithms. The detailed procedure to select this algorithm is as follows:
```
Comment (andymunro) [inline server_admin/topics/clients/client-oidc.adoc]: add "the" before https:

**Review by andymunro — CHANGES_REQUESTED** [review]
one minor change

**Comment by tnorimat** [comment]
@andymunro Thank you for the review. I've revised the document as you suggested.

**Review by mposolda — APPROVED** [review]
@andymunro @tnorimat Thanks for the review and update!

**Pull request merged**